### PR TITLE
Resultformatter

### PR DESCRIPTION
**Pull request by UMTti**

### CODE REVIEW

File: src/main/java/hy/tmc/cli/frontend/ResultInterpreter.java [hunk 1]
```diff
@@ -6,57 +6,41 @@
...
+
+        if (result.status == TESTS_FAILED) {
+            return testFailureReport();
+        } else {
+            return formatter.interpretStatus(result);
         }
-    }
-    
-    private String genericErrorOutput() {
-        return ColorFormatter.coloredString("Failed due to an internal error", RED, WHITE);
```
Comment (jamox) [inline src/main/java/hy/tmc/cli/frontend/ResultInterpreter.java]: Is white the background? should it be transparent

Comment (jamox) [inline src/main/java/hy/tmc/cli/frontend/ResultInterpreter.java]: I mean for cases when the terminal background color is black white background for test might not be nice...

File: src/main/java/hy/tmc/cli/frontend/formatters/CommandLineFormatter.java [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+          case GENERIC_ERROR:
+              return ColorFormatter.coloredString("Failed due to an internal error", RED, WHITE);
+          default:
+              throw new IllegalArgumentException("bad argument");
+        }
+    }
+
+    /**
+     * Some tests failed -explanation
+     * @return
```
Comment (jamox) [inline src/main/java/hy/tmc/cli/frontend/formatters/CommandLineFormatter.java]: No empty return tags, remove these :)

**Comment by samutamm** [comment]
@UMTti Fix commented parts and then merge this pull request to master

**Pull request merged**